mixer/pkg/server/server_test.go

```diff
@@ -24,6 +24,10 @@ import (
 
 	"google.golang.org/grpc"
 
+	"net/http"
```

Comment:
typical "please run bin/fmt.sh" request... (and yes, *this* is the thing that bots should be excellent at ;))

Reply:
I explicitly ran go fmt, but not goimports...